pages/products/index.vue

```diff
@@ -5,7 +5,8 @@ const title = 'premium subscription | rotki';
 const description = 'Features included with a premium rotki subscription';
 
 const config = useRuntimeConfig();
-const baseUrl = config.baseUrl;
+const baseUrl = config.public.baseUrl;
+
 useHead({
```

Comment:
@kelsos 
since `config` is not used on its own, what if we destructure like
```typescript
const { public: { baseUrl } } = useRuntimeConfig();
```

Reply:
True we could also do that